packages/sync-repo-settings/test/test.ts

```diff
@@ -85,7 +89,7 @@ function nockUpdateBranchProtection(
 
 async function receive(org: string, repo: string, cronOrg?: string) {
   await probot.receive({
-    name: 'schedule.repository',
+    name: 'schedule.repository' as any,
```

Comment:
Same as above, I think we can cast this to `'*'`

Reply:
For some reason, in testing, I'm not allowed to cast this to a wildcard. I'll add a link to the issue though.